app/src/main/java/com/github/multimatum_team/multimatum/util/DeadlineSerializer.kt

```diff
@@ -23,6 +23,7 @@ internal class DeadlineSerializer : JsonSerializer<Deadline> {
         obj.addProperty("state", deadline.state.name)
         obj.addProperty("dateTime", formatter.format(deadline.dateTime))
         obj.addProperty("description", deadline.description)
+        obj.addProperty("pdfPath", deadline.pdfPath)
```

Comment:
Are you sure that another user can access files uploaded by an user other than the uploader?